package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "lab-01-modular-patterns",
+  "version": "1.0.0",
+  "description": "401 JS -- Class 01 Lab modular patterns ===",
+  "main": "index.js",
+  "dependencies": {},
+  "devDependencies": {
+    "chai": "^3.5.0",
+    "eslint": "^3.12.2",
+    "gulp": "^3.9.1",
+    "gulp-eslint": "^3.0.1",
+    "gulp-mocha": "^3.0.1",
+    "mocha": "^3.2.0"
+  },
+  "scripts": {
```

Comment:
Great job remembering to add a script, but it's not quite accurate. Here's the script tags you need: 

```
 "scripts": {
    "lint": "./node_modules/eslint/bin/eslint.js .",
    "test": "./node_modules/mocha/bin/mocha"
  },
```

This enables another user who pulls down your code to just type ```npm run test``` or ```npm run lint``` on their terminal without having to globally install Mocha or Lint. In other words, I don't have to type ```mocha``` in my terminal to run your tests, which means I don't need to install it on my end.